dev/tracing_tests/test/common.dart

```diff
@@ -52,3 +54,27 @@ Future<void> runFrame(VoidCallback callback) {
   callback();
   return result;
 }
+
+// This binding skips the zones tests. These tests were written before we
+// verified zones properly, and they have been legacied-in to avoid having
+// to refactor them.
+class TestBinding extends WidgetsFlutterBinding {
```

Comment:
Maybe give this a name that makes people feel bad if they were to use it in a newly written test to avoid that this pattern spreads further? Something like `LegacyTestBinding`, maybe?

Reply:
`ZoneIgnoringTestBinding`?